modules/monitoring-creating-a-monitoringstack-object-for-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-a-monitoringstack-object-for-cluster-observability-operator_{context}"]
+= Creating a MonitoringStack object for the Cluster Observability Operator
+
+To be able to discover the sample `ServiceMonitor` object you created by using the documentation in "Specifying how a service is monitored for Cluster Observability Operator", you must create and configure a `MonitoringStack` object.
+
+You can create a `MonitoringStack` object to discover the sample `ServiceMonitor` object. 
+The `ServiceMonitor` object is for a sample service named `prometheus-coo-example-app` in the `ns1-coo` namespace."
```

Comment:
```suggestion
The `ServiceMonitor` object is for a sample service named `prometheus-coo-example-app` in the `ns1-coo` namespace.
```

Reply:
Is there a way to clarify "is for" here? Maybe we could say something like "The `ServiceMonitor` object monitors the sample service" or be more specific about what it does? Not a blocker or anything, just curious.

Reply:
- Fixed the typo. 
- Excellent idea. Updated per your suggested wording.